### DISCUSSION TITLE
example for blocking a website or ipaddress from accessing from a system?

### DISCUSSION THREAD

**Discussion by rajkumarks7** [corazawaf/coraza#569] [General]
can we use the coraza as a firewall in mac or linux to block certain ipaddress from accessing it from the system.
it is like blocking the outbound of the ipaddress from the system

**Comment by rajkumarks7** [comment]
can you provide a example code to block the outbound rules or inbound rules please. like there are very few examples in the main website or github.

**Comment by jptosso** [comment]
Hey ! You can do the following:

```
SecRule REMOTE_ADDR "1.1.1.1" "id:111,phase:1, msg:'Blocked IP %{TX.0}',deny,status:403"
```
